### PR TITLE
add labels in dev environments docs

### PR DESCRIPTION
**Pull request by AdrianPedriza**
### Proposed changes
documentation about new flags and subcommands to support labels for dev environments.

Changes are:
- new flag `--label` in `okteto pipeline deploy` command
- new flag `--reuse-params` in `okteto pipeline deploy` command
- new command `list` for `okteto pipeline`
- flags supported in `okteto pipeline list` command

### CODE REVIEW

File: src/content/reference/cli.mdx [hunk 1]
```diff
@@ -557,6 +559,26 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 | _--volumes_   |  (bool)  | Destroy the persistent volumes created by the pipeline (defaults to false)                                                               |
 | _--wait_      |  (bool)  | Wait until the pipeline is destroyed (defaults to false)                                                                                 |
 
+#### list
+
+List all your okteto development environments in the namespace in use.
```
Comment (omnomagonz) [inline src/content/reference/cli.mdx:564]:
```suggestion
List all your Okteto development environments in the current namespace.
```

File: src/content/reference/cli.mdx [hunk 2]
```diff
@@ -557,6 +559,26 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
...
+
+```console
+$ okteto pipeline list
+```
+
+Run `okteto pipeline list` to get the status and info of your development environments.
+
+You need to be [logged in](reference/cli.mdx#context) to Okteto before running this command.
```
Comment (omnomagonz) [inline src/content/reference/cli.mdx:572]:
```suggestion
You must be [logged in](reference/cli.mdx#context) to Okteto before running this command.
```

**Review by omnomagonz — APPROVED** [review]
Non-blocking suggestions

File: src/content/reference/cli.mdx (hunk 1 above)
Comment (AdrianPedriza) [reply src/content/reference/cli.mdx]: fd89295ea701b3df76b01406078faafc556997c2

File: src/content/reference/cli.mdx (hunk 2 above)
Comment (AdrianPedriza) [reply src/content/reference/cli.mdx]: I took the opportunity and I also updated other lines with the previously proposed text 019233be7630dbadfa770df3ee4cac41048bc42a

**Pull request merged**